docs/user/explanation/configuration-file.rst

```diff
@@ -0,0 +1,120 @@
+Versioning configuration files
+==============================
```

Comment:
This is the only change to this file, the title was previously "Using a configuration file".